db/schema.rb

```diff
@@ -314,6 +314,28 @@
   add_index "panels_users", ["panel_id"], name: "index_panels_users_on_panel_id", using: :btree
   add_index "panels_users", ["user_id"], name: "index_panels_users_on_user_id", using: :btree
 
+  create_table "posts", force: :cascade do |t|
```

Comment:
@stedmanblake not sure why this schema bit is in here – probably want to check out the schema file from master. reset your db, and run rake db:migrate again to get this out of here?